recipes/cli-duckdb/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cli-duckdb" %}
+{% set version = "0.9.1" %}
+{% set extensions = "httpfs;json;parquet" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/duckdb/duckdb/archive/v{{ version }}.tar.gz
+  sha256: 37a43188d9354ce3ca101b2b118d867f5f76d04c3b83c09d86fd7508351a631b
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - |
+      # echo "duckdb_extension_load(httpfs)" > extension/extension_config_local.cmake
+      make extension_configuration
+      DUCKDB_EXTENSIONS='{{ extensions }}' VCPKG_TOOLCHAIN_PATH="$VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake" make
+
+requirements:
+  host:
+    - cmake
+    - make
+    - vcpkg
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - cmake
+    - make
+    - vcpkg
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - ls -lh
```

Comment:
We should include a better test here. I think it would be sufficient to execute the duckdb cli.